doc/lua_api.txt

```diff
@@ -6266,6 +6266,14 @@ Used by `minetest.register_node`, `minetest.register_craftitem`, and
                          uses = 20, maxlevel = 2},
```

Comment:
Should also mention `punch_attack_uses`

Reply:
> Should also mention punch_attack_uses

This example aims to explain how tools interact with nodes, so I chose not to include it here. (technically `damage_groups` doesn't need to be there either)

> Why do you need the part in register.lua if the group-specific parameter is never used?

The code in register.lua ensures existing tools get an useful `punch_attack_uses` value if they don't specify one themselves.

> group-specific

???
The parameter is part of tool_capabilities, not of one of the groupcaps.

Reply:
Sorry, my mistake. The Lua code actually modifies `toolcaps`, not `cap`, what I erroneously thought.